app/src/main/java/com/framgia/pokemon/screen/main/MainViewModel.java

```diff
@@ -0,0 +1,110 @@
+package com.framgia.pokemon.screen.main;
+
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.support.annotation.NonNull;
+import android.support.design.widget.BottomNavigationView;
+import android.support.v4.app.Fragment;
+import android.view.MenuItem;
+import com.framgia.pokemon.BR;
+import com.framgia.pokemon.R;
+import com.framgia.pokemon.screen.pokemon.PokemonFragment;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by ducpm on 24/08/17.
+ */
+
+public class MainViewModel extends BaseObservable implements MainContract.ViewModel {
+    private static final int TAB_POKEMON = 1;
+    private static final int TAB_FAVORITE = 2;
+    private static final int TAB_REGION = 0;
+    private MainActivity mMainActivity;
+    private MainAdapter mMainAdapter;
+    private MainContract.Presenter mPresenter;
+    private int mPosition;
+    private BottomNavigationView.OnNavigationItemSelectedListener mListener =
+            new BottomNavigationView.OnNavigationItemSelectedListener() {
+
+                @Override
+                public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                    switch (item.getItemId()) {
+                        case R.id.navigation_region:
+                            setPosition(TAB_REGION);
+                            break;
+                        case R.id.navigation_pokemon:
```

Comment:
ERROR: 'case' child have incorrect indentation level 24, expected level should be one of the following: 16, 20.